server/src/instrument/provider/display.ts

```diff
@@ -136,6 +147,21 @@ This command waits until a user responds to a front‑panel prompt that was crea
         kind: CompletionItemKind.Function,
         label: 'waitevent',
     },
+    {
+        data: { domains: ['display'] },
+        documentation: {
+            kind: MarkupKind.Markdown,
+            value: '```lua\ndisplay.watchchannels\n```\n\
+\n\
+Get or set which channels are set to be watched from the front panel, you can define up to 20.\n\
+\n\
+Watch Channels affect what you see on the scan and statistics swipe screens. In the Reading Table, you can select \
+the watch channels to filter the buffer so that only data from the watch channels is shown. In the Graph screen, \
+you can select the watch channels as the traces to display on the graph.'
```

Comment:
After mentioning a complicated Proper-Noun idea in a completion (e.g. Channel Name), there are two options:
1. we explain the entire idea (see the `status` namespace) if the concept is isolated.
2. we tell the user to look in the manual for more information if the concept cross-namespace.

In this instance, option 2 would be better, so stab a line at the very end:
> ...on the graph.
>
> Refer to the manual for more information on Channel Names.

Reply:
For functions that use the channelList/channelNumber convention, do you think it would be appropriate to use the signature documentation to point towards the Reference Manual?  

Obviously (?) for a property like `display.watchchannels` that can’t happen, but most of the `channel` namespace uses this convention in functions.